csharp/generate/templates/api.mustache

```diff
@@ -1,38 +1,38 @@
 {{>partial_header}}
+
 using System;
 using System.Collections.Generic;
 using System.Collections.ObjectModel;
 using System.Linq;
-{{#netStandard}}
-using RestSharp.Portable;
-{{/netStandard}}
-{{^netStandard}}
-using RestSharp;
-{{/netStandard}}
+using System.Net;
```

Comment:
I am assuming that the differences shown below are due to the upgrade to the new version of the OpenApi generator?

Reply:
yeh - I've copied and then updated the latest templates from the Open API generator repo